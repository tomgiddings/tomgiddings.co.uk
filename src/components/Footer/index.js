import React from 'react';

function Footer() {
  return (
    <footer>Designed by Tom Giddings and built in React.js. <a href="https://github.com/tomgiddings/tomgiddings">View the code on Github</a>.</footer>
  );
}
export default Footer;
