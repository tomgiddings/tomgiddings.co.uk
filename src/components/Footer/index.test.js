import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Footer from './';

afterEach(cleanup);

it('Renders correctly', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
