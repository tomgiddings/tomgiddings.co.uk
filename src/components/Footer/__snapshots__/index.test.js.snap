// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<DocumentFragment>
  <footer>
    Designed and built by Tom Giddings using React. 
    <a
      href="https://github.com/tomgiddings/tomgiddings"
    >
      View the code on Github
    </a>
    .
  </footer>
</DocumentFragment>
`;
